packages/gatsby-transformer-screenshot/lambda/index.js

```diff
@@ -37,13 +39,13 @@ exports.handler = async (event, context, callback) => {
     })
 }
 
-exports.run = async (browser, url, width, height) => {
+exports.run = async (browser, url, width, height, fullPage) => {
   console.log(`Invoked: ${url} (${width}x${height})`)
```

Comment:
What do you think about adding fullPage to the log output? 

```
console.log(`Invoked: ${url} (${width}x${height}) fullPage:${fullPage}`)
```

Would it be useful or just unnecessary noise?

Reply:
I did notice that. Didn't add fullPage to it because it is not the part of keyBase, which is the important part of the function that would be useful debugging purpose.